networks/mainnet/rpcs.md

```diff
@@ -4,48 +4,170 @@ description: Public RPC endpoints are available for the Filecoin mainnet.
 
 # RPCs
 
-{% hint style="info" %}
-[Chainlist](https://chainlist.org/?search=filecoin&testnets=true) contains a dynamically updated list of available Filecoin RPCs. [Find out more at chainlist.org's Filecoin listings](https://chainlist.org/?search=filecoin&testnets=true).
-{% endhint %}
-
-These endpoints are limited to [read-only Filecoin JSON RPC API calls](../../reference/json-rpc/) and [`MPoolPush`](../../reference/json-rpc/mpool.md) for sending messages that have already been signed.
-
-## [Ankr](https://ankr.com)
-
-* HTTPS: `https://rpc.ankr.com/filecoin`
-* [Supported Filecoin API methods](https://www.ankr.com/docs/rpc-service/chains/chains-list/#filecoin)
-
-## [ChainupCloud](https://cloud.chainup.com)
-
-* HTTPS: `https://filecoin.chainup.net/rpc/v1`
-* WebSocket: `wss://filecoin.chainup.net/rpc/v1`
-* [ChainupCloud documentation](https://docs.chainupcloud.com/blockchain-api/filecoin/public-apis)
-
-## [Glif](https://api.node.glif.io)
-
-Please note that publicly available hosted endpoints **only guarantee 2000 of the latest blocks.**
-
-* HTTPS: `https://api.node.glif.io/rpc/v1`
-* WebSocket: `wss://wss.node.glif.io/apigw/lotus/rpc/v1`
-*   Lotus lite-node command:
+[Chainlist](https://chainlist.org/?search=filecoin&testnets=true) provides a dynamically updated list of [available Filecoin - Mainnet RPCs](https://chainlist.org/?search=filecoin&testnets=true).
+
+These endpoints are limited to [read-only Filecoin JSON RPC API calls](../../reference/json-rpc/) including Filecoin Eth RPC methods and [`MPoolPush`](../../reference/json-rpc/mpool.md) for sending already signed messages.
```

Comment:
```suggestion
These endpoints are limited to [read-only Filecoin JSON RPC API calls](../../reference/json-rpc/) including [Filecoin Eth RPC methods](https://docs.filecoin.io/reference/json-rpc/eth).  The only write operation is [`MPoolPush`](../../reference/json-rpc/mpool.md#mpoolpush) for sending already signed messages.
```

Reply:
@BigLep - technically the Eth RPC also has a write operation, `eth_sendRawTransaction`

Reply:
Ack.  I updated the suggested change.